src/course/handbook/projects.md

```diff
@@ -1,75 +1,71 @@
-6. Avoid talking through the project brief for a long time - each team has the same brief and your time will be better spent talking about what was unique about your presentation
+A good github workflow is essential to keeping a project organised and easy to maintain. The larger the project, the more important it becomes to track changes and be aware of who worked on what.
 
-### Preparing for presenting
+Some things to keep in mind:
 
-Your team should spend some time preparing what you'll present. Alongside creating a slide deck, this means preparing which parts of the presentation you'll each talk through and ensuring your presentation fits within the time available.
+- Make sure everyone has the same code formatter ([prettier](https://prettier.io/docs/en/install.html)) setup, so you can avoid merge conflicts due to weird spaces/extra commas etc.
+- Never work on the main branch. Before you start work, make sure you pull main and create a new branch.
 
-Decide together which sections each person will talk through. Who will share the screen for the demo? Who will talk and who will click through? Choose someone on your team to keep time for the team.
+```git
+   git pull
+   git checkout -b "new-feature"
+```
 
-Spend some time revisiting your codebase and understanding the code. Are there sections which you couldn't talk about? If so, spend time reading through the code **together** to ensure you are all on the same page, especially for code you didn't write yourself. Everyone is responsible for this.
+- Commit often. This makes it easier to undo specific changes. If you've been coding for half a day and forgot to commit, VSCode lets you [stage selected lines](https://stackoverflow.com/questions/34730585/how-can-i-commit-some-changes-to-a-file-but-not-others-in-vscode) from the source control panel.
+- Write meaningful commit messages. A good convention to follow for commit messages is that they should be imperative, i.e: "add tests for signup form"
 
-### Make it interactive
+- Your commit messages should make sense if read in this format:
+  `"If applied, this commit will: <your commit message>"`
 
-- **Ask your audience questions** and keep them on their toes.
-- You could even introduce a quiz element to your presentation. Check out [Kahoot!](https://kahoot.com)
+- When you pair, don't forget to co-commit! (you can find more information in the [GitHub Docs](https://docs.github.com/en/github/committing-changes-to-your-project/creating-and-editing-commits/creating-a-commit-with-multiple-authors))
 
-### Include code snippets
+- When you finish working on your branch, make sure it's upto date with main before pushing
 
-- But keep them concise!
-- We want to hear about something novel in your project, give the audience an understanding - without overwhelming them with line by line detail.
+```git
+   git checkout main
+   git pull
+   git checkout "new-feature"
+   git merge main
+   git push
```

Comment:
This can be simplified to just
```suggestion
   git pull origin main
```
without changing branches. It'll fetch the remote `main` and merge them into your current branch